requirements.txt

```diff
@@ -1,3 +1,7 @@
 requests==2.27.1
 path==16.4.0
-python-dotenv==0.19.2
\ No newline at end of file
+python-dotenv==0.19.2
+jina-hubble-sdk==0.1.3.dev8
+pytest==7.0.0
```

Comment:
make sure only "mandantory" libraries in `requirements.txt`, others move to `requirements-dev` or `requirements-extra.txt`